cfpb/consumerfinance.gov
# Increase timeout for Deploy Stack stage to 30 minutes

15 minutes seems to usually not be enough when refreshing the DB and doing the ES indexing.


## How to test this PR

It should be self-testing! Keep an eye on https://[zusa]/job/consumerfinance.gov/job/PR-6001/. Almost every PR fails its first deployment because Deploy Stack takes too long. Let's see if this one works on the first try.


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
